tshirts.c

```diff
@@ -14,9 +16,25 @@ char size(int cms) {
 }
 
 int main() {
-    assert(size(37) == 'S');
-    assert(size(40) == 'M');
-    assert(size(43) == 'L');
+    //for (int i = 30; i < 64; i++)
+    int i;
+    time_t t;
+    srand((unsigned) time(&t));
+    while(1)
+    {
+        i =(rand()%(48-29)) + 29;

```

Comment:
using random function to generate a test input value will always lead to non deterministic test coverage.
It may so happen that values generated might not be of size 's' at all or on similar lines for other sizes.
Use discrete values with boundary values, in range values and out of range values.